exec/cnex/lib/process_win32.c

```diff
@@ -9,6 +9,25 @@
 #include "nstring.h"
 
 
+struct ThreadInfo {
+    HANDLE pHandle;
```

Comment:
Please don't use this weird naming notation. If this stands for "pipe handle", then spell it out.

Reply:
Changed naming convention in both places.